CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [#5980](https://github.com/rubocop-hq/rubocop/issues/5980): Add --safe and --safe-auto-correct options. ([@Darhazer][])
 * [#4156](https://github.com/rubocop-hq/rubocop/issues/4156): Add command line option `--auto-gen-only-exclude`. ([@Ana06][], [@jonas054][])
+* Add new `Naming/ForbidenVariableName` cop. Forbide the use of variable names and recommend alternatives if available. ([@ylecuyer][])
```

Comment:
Be careful you miss type:
Add new `Naming/ForbiddenVariableName` cop. Forbid the use of variable names and recommend alternatives if available.